File: lib/models/imin_scale_obj.dart
class IminScaleObj{
  final String weight;
  final String weightStatus;

  IminScaleObj(this.weight, this.weightStatus);

  factory IminScaleObj.fromJson(Map<String, dynamic> json) {
    return IminScaleObj(json['weight'], json['weightStatus']);
  }
}

File: android/src/main/kotlin/com/ezyorder/flutter_imin_scale/models/IminScaleObj.kt
package com.ezyorder.flutter_imin_scale.models

class IminScaleObj(val weight: String, val weightStatus: String) {
    companion object {
        fun fromJson(json: Map<String, Any>): IminScaleObj {
            return IminScaleObj(
                json["weight"] as String,
                json["weightStatus"] as String
            )
        }
    }
}